### DISCUSSION TITLE
2015/04/21/salt-lake-city-to-boise-beacon-27

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#16] [Announcements]
# 2015/04/21/salt-lake-city-to-boise-beacon-27

http://localhost:4000/2015/04/21/salt-lake-city-to-boise-beacon-27

**Comment by jengalas** [comment]
Posted by Nancy Ray
July 31, 2023 at 3:23 pm
_Imported from WordPress_

I now live on the property at 742 Desert Wind Road, Boise 83716 Aka Mayfield. I live on the corner of pie shaped lot bordering Regina Road and Desert Wind Road, which was the Old Highway 30. When the weather cools off I will look for the marker. The arrow is in sad shape. My intent upon moving here was too keep it cleaned up. Unfortunately I had an accident the first year and have not been able to maintain it.
